ios/FluentUI/ActivityIndicator/ActivityIndicatorTokenSet.swift

```diff
@@ -15,6 +15,23 @@ import UIKit
     case xLarge
 }
 
+extension MSFActivityIndicatorSize : CustomStringConvertible {
+     public var description: String {
+         switch self {
+         case .xSmall:
+             return "xSmall"
+         case .small:
+             return "small"
+         case .medium:
+             return "medium"
+         case .large:
+             return "large"
+         case .xLarge:
+             return "xLarge"
+         }
+     }
+ }
+
```

Comment:
I don't think this extension should be showing up in release builds. If possible, we should find a way to add the functionality you need with 0 increase to the lib.